tables/eo-g1.ctb

```diff
@@ -34,56 +34,109 @@
 # October 13, 2013
 
 include spaces.uti
-punctuation \x00a0 0		NO-BREAK SPACE
-
-include braille-patterns.cti
-
-numsign 3456  number sign
-capsletter 46
-begcapsword 46-46
-endcapsword 46-3
+ 
 punctuation ! 235		EXCLAMATION MARK
 punctuation ' 3		APOSTROPHE
 punctuation ( 2356		LEFT PARENTHESIS
 punctuation ) 2356		RIGHT PARENTHESIS
-punctuation [  2356-2356
-punctuation ] 2356-2356
 punctuation , 2		COMMA
 punctuation - 36		HYPHEN-MINUS
-punctuation — 6-36
 punctuation . 256		FULL STOP
 punctuation : 25		COLON
 punctuation ; 23		SEMICOLON
 punctuation ? 26		QUESTION MARK
 punctuation " 236
-punctuation … 3-3-3
 punctuation / 34
 punctuation * 35
-prepunc " 236
-postpunc " 356
 
-# Overwrite the definition of w from latinLetterDef6Dots.uti. For the
-# table to work this rule needs to be defined before AND after the
-# include.
-lowercase w 23456
-
-include latinLetterDef6Dots.uti
-
-lowercase w 23456
+# Don't include latinLetterDef6Dots.uti
+# In theory, the exceptions can be handled, but the risk is greater.
```

Comment:
I think it would be nice to keep including latinLetterDef6Dots.uti if possible. Is there really a good reason to not include it?

Reply:
> Some Latin letters are defined differently. So if we were to include latinLetterDef6Dots.uti, some letters would have to be defined out of order, and the definition in latinLetterDef6Dots.uti would be left hanging, which I don’t really like.

How many letter are defined differently?

Reply:
Sorry, I forgot. It's actually only one letter, "w", but I still don't like including a file, depending around the problems that implies. If you think it is necessary, you are welcome to change it.

Reply:
In such cases I think it's fine to include a file and overwrite the one definition. Since https://github.com/liblouis/liblouis/pull/1481 there should not be any risk anymore. And tests should ensure the table works as expected anyway.